packages/compadre/examples/GMLS_Device.cpp

```diff
@@ -263,7 +197,7 @@ bool all_passed = true;
     // CreatePointCloudSearch constructs an object of type PointCloudSearch, but deduces the templates for you
     auto point_cloud_search(CreatePointCloudSearch(source_coords, dimension));
 
-    double epsilon_multiplier = 1.5;
+    double epsilon_multiplier = 1.4;
```

Comment:
Previously, the behavior for convergence on a solution where the solution lies within the span of the GMLS basis is machine precision, until it isn't. At that point the error becomes arbitrarily large. Because of the pivoting in the QR algorithm that we are now using, when we don't have enough neighbors for unisolvency, the error grows gracefully and as a function of how many neighbors are missing. 1.4 is sufficient to stay within the tolerance specified in the example, so we might as well use it since it runs faster.